src/components/Thumbnail.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+
+export const Thumbnail = () => {
+  return <Wrapper></Wrapper>;
```

Comment:
내부에 사용하는 요소가 없는 경우 self-closing tag를 활용해보시면 좋겠네요
```suggestion
  return <Wrapper />
```

Reply:
넵 수정하도록 하겠습니다!